### PR TITLE
Update for compatibility with google-fluentd 1.6.25+

### PR DESCRIPTION
**Pull request by jpatokal**
- Creates a new fluentd agent config to override incompatible default (fixes issue #18 )
- Update to latest versions of stackdriver-logging-agent, prometheus-to-sd
- Removes unnecessary nodeSelector that does not match GKE defaults

### CODE REVIEW

File: kubernetes/fluentd-configmap.yaml [hunk 1]
```diff
@@ -171,135 +171,6 @@ data:
         process_start_timestamp ${record["process_start_timestamp"].to_i}
       </record>
     </filter>
-  output.conf: |-
```
Comment (igorpeshansky) [inline kubernetes/fluentd-configmap.yaml:174]: Note that you could add multiple configmap objects to the same YAML file by using the `---` separator, e.g.:
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: fluentd-gcp-config
…
data:
  containers.input.conf: |-
    …
---
kind: ConfigMap
apiVersion: v1
  name: fluentd-gcp-main-config
…
data:
  google-fluentd.conf: |-
    …
```
That might make things simpler…

**Review by igorpeshansky — CHANGES_REQUESTED** [review]
Should probably mention that this fixes #18…

File: kubernetes/fluentd-configmap.yaml (hunk 1 above)
Comment (jpatokal) [reply kubernetes/fluentd-configmap.yaml:174]: Thanks, merged the files together and successfully tested, PTAL.

**Review by igorpeshansky — COMMENTED** [review]
Sorry for the delay. LGTM :shipit:

**Pull request merged**